### DISCUSSION TITLE
Hpool incorrect information

### DISCUSSION THREAD

**Discussion by sunyao920807** [Chia-Network/chia-blockchain#4394] [Show and tell]
There are 4608 blocks in the whole network. The data of hpool pool is 1355p, and 2779p accounts for 48%. The data that should be disclosed is 2247. The data that hpool publishes is 1232, which steals half of the revenue. The announced revenue is reduced by 25%, and the actual revenue is reduced by 40%

**Comment by SackofFluor** [comment]
So hpool is ripping people off?

**Reply by sunyao920807** [reply]
must be

**Comment by andrejreimer** [comment]
That's true, i checked the numbers. Hpool keeps nearly half of the revenue.

**Reply by SackofFluor** [reply]
God... I am so happy that I didn't fall for hpool wenn I started plotting....
Well on the other side I still haven't gotten anything and don't even know whether everything is actually working 😅😅

**Comment by andrejreimer** [comment]
Still better than getting nothing. I don't see why not everybody should join hpool or uupool. As you need to replot for poolfarming anyway. That way you could get back at least a little bit...

**Reply by SackofFluor** [reply]
Well, that is true... but I am waiting until there is a more official pool protocol before I start replotting. I will do the replotting then step by step and not just delete everything and start from scratch...

But I don't know, taking half the profits is pretty crazy, don't you think?

**Comment by andrejreimer** [comment]
It is super crazy and the founders are to blame , they don't really care about the little farmers. I have plotted nearly 300 Plots when they anounced that i would need to replot, officially. I hope to compensate some of the weardown of my ssds. When plotting for pools will be available I will start to replace hpool with the new pool plots. That way i should earn a few hundreds till proper pools arrive.

**Reply by sunyao920807** [reply]
We are not farmers, we are slaves in Hpool
